eat-chips.html

```diff
@@ -1,131 +1,153 @@
-      <li class="bold monospace"><br><h6>S H O P  &nbsp &nbsp H E R E </h6><i class="material-icons">shopping_cart</i></li>
+      <li class="teal lighten-3"><a href="index.html" class="waves-effect waves-teal">HOME</a></li>
+      <li class="space"><pre>S H O P H E R E<i class="material-icons">shopping_cart</i></pre></li>
       <li class="no-padding teal lighten-3">
           <ul class="collapsible collapsible-accordion teal lighten-3">
-              <li class="bold z-depth-1"><a class="collapsible-header coll active waves-effect waves"><b>E a t a b l e s</b></a>
-                <div class="collapsible-body">
-                <ul>
-                  <li class="cyan lighten-5"><a href="shop.html" class="active">&nbsp Chips</a></li>
-                  <li><a href="tc.html">&nbsp Chocolates</a></li>
-                  <li><a href="snack.html">&nbsp Daily Snacks</a></li>
-                  <li><a href="bisc.html">&nbsp Biscuits</a></li>
-                  <li><a href="ration.html">&nbsp Ration</a></li>
-                  <li><a href="pack.html">&nbsp Packed Stuff</a></li>
-                  <li><a href="other.html">&nbsp others</a></li>
+              <li  class="space" style="padding-top:10px;"><a class="collapsible-header active waves-effect waves">Eatables</a>
+                <ul class="collapsible-body">
+                  <li class="cyan lighten-5 coll"><a href="shop.html" class="active">Chips</a></li>
+                  <li><a href="tc.html">Chocolates</a></li>
+                  <li><a href="snack.html">Daily Snacks</a></li>
+                  <li><a href="bisc.html">Biscuits</a></li>
+                  <li><a href="ration.html">Ration</a></li>
+                  <li><a href="pack.html">Packed Stuff</a></li>
+                  <li><a href="other.html">others</a></li>
                 </ul>
-                </div>
               </li>
   <br>
-                <li class="bold z-depth-1" ><a class="collapsible-header coll waves-effect waves-teal"><b>S t a t i o n a r y</b></a>
+                <li class="space" style="padding-top:10px;"><a class="collapsible-header waves-effect waves-teal">Stationary</a>
                 <div class="collapsible-body">
                 <ul>
-                  <li><a href="chip.html">&nbsp Noting stuff</a></li>
-                  <li><a href="tc.html">&nbsp Pencil&Pen</a></li>
-                  <li><a href="snack.html">&nbsp Other items </a></li>
+                  <li><a href="chip.html">Noting stuff</a></li>
+                  <li><a href="tc.html">Pencil Pen</a></li>
+                  <li><a href="snack.html">Other items </a></li>
                 </ul>
                 </div>
               </li>
   <br>
-              <li class="bold z-depth-1"><a class="collapsible-header coll waves-effect waves-teal"><b>D o m e s t i c&nbsp I t e m s</b></a>
+              <li class="space" style="padding-top:10px;"><a class="collapsible-header waves-effect waves-teal">Domestic Items</a>
```

Comment:
Avoid inline CSS. Instead define in the form of Classes to make it more general and organised.